src/components/hooks/useVisibility.tsx

```diff
@@ -53,25 +53,27 @@ export const useVisibility = (item: PackedItem): UseVisibility => {
   const [isDisabled, setIsDisabled] = useState(false);
   useEffect(() => {
     // disable setting if any visiblity is set on any parent items
-    setIsDisabled(
-      Boolean(
-        (itemLoginSchema && itemLoginSchema?.item?.path !== item?.path) ||
-          (item?.public && item?.public?.item?.path !== item?.path),
-      ),
-    );
+
+    const isItemLoginDisabled =
+      itemLoginSchema &&
+      itemLoginSchema?.status !== ItemLoginSchemaStatus.Disabled &&
+      itemLoginSchema?.item?.path !== item?.path;
```

Comment:
Not sure but I think this is wrong. `isItemLogingDisabled` should have `itemLoginSchema`'s status to `Disabled` right?

Reply:
Actually this is a variable to disable the option if you selected it (and are not at the right place to edit it). ItemLoginSchema should be set to be disabled, so it means the schema shouldn't be disabled. It changed with the refactor since now we're not deleting it anymore. I will add comments.